core/system/src/unvic.c

```diff
@@ -583,6 +583,22 @@ void unvic_init(void)
      * levels. */
     assert(__UVISOR_NVIC_MIN_PRIORITY < UVISOR_VIRQ_MAX_PRIORITY);
 
+    /* Set the priority of each exception. SVC is lower priority than
+     * MemManage, BusFault, and UsageFault, so that we can recovery from
+     * stacking MemManage faults more simply. Set DebugMonitor, PendSV, SYSTICK
+     * to the same priority as SVC. */
```

Comment:
The comment `Set DebugMonitor, PendSV, SYSTICK to the same priority as SVC.` is outdated now.